web/styles/components.css

```diff
@@ -31,15 +31,18 @@
                the light-mode color value here as a fallback. */
             border: 1px solid
                 var(--color-border-rendered-checkbox, hsl(0deg 0% 0% / 60%));
-
             border-radius: 4px;
-            filter: brightness(0.8);
 
             cursor: pointer;
         }
 
         &:checked ~ .rendered-checkbox {
-            background-image: url("../images/checkbox.svg");
+            /* As with the border color above, we duplicate
+               the light-mode SVG URL value here as a fallback. */
+            background-image: var(
+                --svg-url-rendered-checkbox,
+                url("../images/checkbox.svg")
+            );
```

Comment:
Did you test that the portico signup pages are unchanged?

Reply:
Yes. Confirmed that the fallback SVG renders, as CSS variables are not brought into the portico:

![Screenshot 2024-09-16 at 5 20 28 PM](https://github.com/user-attachments/assets/2cbb9cc3-1908-4fcb-932c-6908ec26f4f6)